opentech/apply/funds/models.py

```diff
@@ -412,14 +430,32 @@ def build_json_order_by(field):
         return super().order_by(*field_ordering)
 
 
-class ApplicationSubmission(AbstractFormSubmission):
+class ApplicationSubmission(WorkflowHelpers, AbstractFormSubmission):
     form_data = JSONField(encoder=DjangoJSONEncoder)
     page = models.ForeignKey('wagtailcore.Page', on_delete=models.PROTECT)
     round = models.ForeignKey('wagtailcore.Page', on_delete=models.PROTECT, related_name='submissions', null=True)
     user = models.ForeignKey(settings.AUTH_USER_MODEL, on_delete=models.SET_NULL, null=True)
 
+    # Workflow inherited from WorkflowHelpers
+    status = models.CharField(max_length=254)
+    status_name = models.CharField(max_length=50)
```

Comment:
Should this be an attribute? It is a display value. `def get_status_display` ?

Reply:
I was wondering about this overnight. The reason I added the `status_name` was because of querying/filtering. We will likely have the same `status_name` e.g. "in review" but vastly different status e.g:
`<workflow 1>__<stage 5>__in_review`
`<workflow 2>__<stage 1>__in_review`

So i guess partly premature optimising. I have got some ideas about alternatives to this that i could try when actually implementing filters so happy to remove.

Reply:
That makes sense, but let's make the change when it is needed ;)